iothub/device/src/Exceptions/IotHubClientException.cs

```diff
@@ -185,9 +195,18 @@ internal IotHubClientException(bool isTransient, IotHubStatusCode statusCode) :
         public string TrackingId { get; set; }
 
         /// <summary>
-        /// The HTTP status statusCode returned by the IoT hub service.
+        /// The HTTP status code.
+        /// </summary>
+        /// <remarks>
+        /// This property is not actually obtained from the response, but mapped from the property <see cref="ErrorCode"/> with the best effort.
+        /// For more details, check <see cref="httpStatusCodes"/>.
```

Comment:
Don't link to things not in the public API surface otherwise the link won't go anywhere.
```suggestion
```